server/src/server.ts

```diff
@@ -4,7 +4,8 @@ import { api, mongo } from './conf/development.json';
 
 const API_SERVER_PORT = api.port;
 
-const DB_URI = `mongodb://${mongo.host}:${mongo.port}/${mongo.db}`;
+// const DB_URI = `mongodb://${mongo.host}:${mongo.port}/${mongo.db}`;
```

Comment:
Import this data from the json file instead of hardcoding

Reply:
Done with changes